components/scream/scripts/nc_file_init.py

```diff
@@ -0,0 +1,47 @@
+from utils import expect
+
+from netCDF4 import Dataset
+
+import pathlib
+
+###########################################################################
+def nc_file_init(output_file,ne=0,np=0,ncol=0,nlev=0,phys_grid="gll",overwrite=False):
+###########################################################################
+
+    expect (phys_grid=="gll", "Error! So far, only GLL physics grid supported.")
+
+    ofile = pathlib.Path(output_file).resolve().absolute()
+    expect ( (not ofile.exists()) or overwrite,
+            "Error! Output file '{}' already exists. Use -o to overwrite.".format(ofile))
+
+    ds = Dataset(ofile,"w",format="NETCDF4",persist=True)
+
```

Comment:
A weird point, but E3SM (and I think SCORPIO and therefore SCREAM) only supports netcdf3. I think we probably want 'NETCDF3_64BIT_OFFSET' but @wlin7 or @brhillman may remember differently.

Reply:
Interesting, I didn't know that.